client/components/migration/migration-runner.js

```diff
@@ -180,7 +180,7 @@ const migrationStateTransitions = {
     stateDone: { // Done state.
         success: null,
         fail: null,
-        callback: null
+        callback: stateErrorHandlingAPICall( MIGRATION_STATE_ENUM.COMPLETED )
```

Comment:
WCS&T is no longer deactivated. `WC_REST_Connect_Migration_Flag_Controller controller` and its `wc/v1/connect/migration-flag` will work. We no longer need to rely on `plugin_deactivation` to update the "completed state". We will update the status here instead.